.gitignore

```diff
@@ -1 +1,4 @@
 /Manifest.toml
+sentencepiece_model.proto
```

Comment:
Do you have a good solution to packaging these autogenerated files? c.f. #1 - I checked them in because I didn't want to deal with building them at run time, and it looks like the protobuf interface has been stable for years. At this point I think it would be easier to check them in as proper static files and not gitignore them.

Reply:
No, I haven't worked with protobuf before. Makes sense to me to check them in though, since I guess if we were compiling a binary like in C or rust the code wouldn't be dynamic there either.

Reply:
I've checked them in in commit e04cc3e8e9915bc878a3a77bf142ecabc5a0e2ff. I double-checked everything worked Ok w/ a fresh install:

```julia
(@v1.9) pkg> activate --temp
  Activating new project at `/var/folders/jb/plyyfc_d2bz195_0rc0n_zcw0000gp/T/jl_yBKxdd`

(jl_yBKxdd) pkg> add https://github.com/ericphanson/LanguageModels.jl#eph/mmap
...
(jl_yBKxdd) pkg> test LanguageModels
...
Test Summary:     | Pass  Total  Time
LanguageModels.jl |    3      3  6.3s
```